viamonvif/service.go

```diff
@@ -319,6 +323,14 @@ func downloadPreviewImage(ctx context.Context, logger logging.Logger, snapshotUR
 	return dataURL, nil
 }
 
+func isImageMIMEType(mimeType string) bool {
```

Comment:
Do we expect to use this helper elsewhere in the future? If no, maybe we should just keep the logic in-line

Reply:
Sounds good to me, makes it easier to read without jumping around.